.github/ISSUE_TEMPLATE/feature.yaml

```diff
@@ -0,0 +1,18 @@
+name: 🚀 Feature request
+description: Suggest an idea for this project.
+labels: ["feature"]
```

Comment:
This label doesn't actually exist. Should change to

```suggestion
labels: ["enhancement"]
```

Reply:
✅ @adlerjohn